mlperf_logging/compliance_checker/training_4.0.0/closed_common.yaml

```diff
@@ -0,0 +1,11 @@
+
+- KEY:
+    NAME:  submission_benchmark
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] in ['resnet', 'ssd', 'stable_diffusion', 'maskrcnn', 'gpt3', 'dlrm_dcnv2', 'bert', 'rnnt', 'unet3d', 'gnn'] "
+    POST:  " enqueue_config('training_4.0.0/closed_{}.yaml'.format(v['value'])) "
+
+- KEY:
+    NAME: gradient_accumulation_steps
```

Comment:
We actually do not have this key in the reference implementation (`mllog.constants.GRADIENT_ACCUMULATION_STEPS`).  Should we also include this in the reference branch? Or is it okay for us to directly remove this in the `closed_common.yaml` checker?

Reply:
Let's include gradient acccumulation in reference, as we should not be modifying the common yaml.

Reply:
Gradient accumulation is included in reference.